matlab/doc/matlab_interface_for_apache_arrow_design.md

```diff
@@ -258,7 +258,7 @@ For large tables used in a multi-process "data processing pipeline", a user coul
 ## Testing 
 To ensure code quality, we would like to include the following testing infrastructure, at a minimum: 
 1. C++ APIs 
-   - GoogleTest C++ Unit Tests 
+   - C++ Unit Tests
```

Comment:
Can we remove this item?

Reply:
That's a good question.

I think it's fine to remove this for the time being (or, at least reword this section to talk about the MEX-based C++ testing approach).

The thinking was that we might want to add some C++ tests in the future. However, we haven't yet had a compelling need to do that, so I think it makes sense to change the wording here.

Reply:
I restructured the testing section slightly and added a note about testing internal C++ code via MEX function calls from a MATLAB Class-Based Unit test.

Reply:
Marking this as resolved.